cmd/teleirc.go

```diff
@@ -18,42 +17,43 @@ const (
 
 var (
 	flagPath    = flag.String("conf", ".env", "config file")
-	flagDebug   = flag.Bool("debug", false, "enable debugging")
+	flagDebug   = flag.Bool("debug", false, "disable debugging")
 	flagVersion = flag.Bool("version", false, "displays current version of TeleIRC")
 )
 
 func main() {
 	flag.Parse()
 
+	verbose := internal.Debug { *flagDebug	}
```

Comment:
Could you change this to reference the field name? Not a huge deal, but it's a linting error for me.
```go
verbose := internal.Debug{DebugLevel: *flagDebug}
```

Reply:
Sure. I already did it.